var/spack/repos/builtin/packages/py-lightning/package.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyLightning(PythonPackage):
+    """Use Lightning Apps to build everything from production-ready,
+    multi-cloud ML systems to simple research demos.
+    """
+
+    homepage = "https://github.com/Lightning-AI/lightning"
+    pypi = "lightning/lightning-2.0.0.tar.gz"
+
+    maintainers("adamjstewart")
```

Comment:
This is more explicit. I might get too busy and remove myself from the base class, but I wouldn't remove myself from this because I actually use it in my research. I try to explicitly add myself to all ML/geo packages I actually use and care about.